### PR TITLE
Refactor all list views (cards and lines): infinite scroll and contextual search

### PR DESCRIPTION
**Pull request by SamuelHassine**
This PR is about the issues #151 and #158.

**Frontend**

- Use generic and common components for infinite scroll (`ListCards` and `ListLines`).
- Migrate all entities list views to these components.
- Introduce the `search` parameter in all fragments and use it for contextual search.
- Delete all "exports" buttons waiting for the new export architecture.
- Delete duplicate codes

**Backend**

- Introducing missing Grakn search helper for entities not indexed in ES (Group, KillChainPhases, etc.)
- Fix a bug in the domain `user.js` when deleting a user that does not have any token (ie. a _person_).

### CODE REVIEW

File: opencti-platform/opencti-graphql/src/database/grakn.js [hunk 1]
```diff
@@ -878,6 +878,7 @@ export const paginate = (
   computeCount = true
 ) => {
   try {
+    console.log(query);
```
Comment (richard-julien) [inline opencti-platform/opencti-graphql/src/database/grakn.js]: To remove

File: opencti-platform/opencti-graphql/src/domain/stixObservable.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ export const findAll = args => {
 export const stixObservablesNumber = args => ({
   count: countEntities('stix_observables', args),
   total: countEntities('stix_observables', dissoc('endDate', args))
-  /*count: getSingleValueNumber(
+  /* count: getSingleValueNumber(
```
Comment (richard-julien) [inline opencti-platform/opencti-graphql/src/domain/stixObservable.js:65]: To remove

File: opencti-platform/opencti-graphql/src/database/grakn.js (hunk 1 above)
Comment (SamuelHassine) [reply opencti-platform/opencti-graphql/src/database/grakn.js]: Done.

File: opencti-platform/opencti-graphql/src/domain/stixObservable.js (hunk 2 above)
Comment (SamuelHassine) [reply opencti-platform/opencti-graphql/src/domain/stixObservable.js:65]: Done, and remove all commented Grakn code.

**Pull request merged**